tedana/workflows/tedana.py

```diff
@@ -463,7 +463,7 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
         io.filewrite(s0_full, op.join(out_dir, 's0vG.nii'), ref_img)
 
     # optimally combine data
-    data_oc = combine.make_optcom(catd, tes, mask, t2s=t2s_full, combmode=combmode)
+    data_oc = combine.make_optcom(catd, tes, masksum, t2s=t2s_full, combmode=combmode)
```

Comment:
`mask`, `masksum`, and `adaptive_mask` are used inconsistently throughout the code. Here is one example where masksum in the input and make_optcom calls it adaptive_mask.
My suggestion is that `mask` should always refer to a boolean mask. `adaptive_mask` should refer to what is sometimes called `masksum`. `masksum` is a confusing variable name and should be removed.

Reply:
Totally agreed, but I think that renaming the variable throughout the package is outside the scope of this PR. In this case, I'm only editing `tedana.py` as much as I have to, but we do have #448, which I think should have its own PR that addresses this.

Reply:
Sounds good. While we're editing `make_adaptive_mask` I could see making the name change just within that function, but that function-specific change can also be done as part of #491 where getting consistent names right will be more important within a few sub-functions. I'll resolve this critique for now.